src/index.ts

```diff
@@ -567,8 +580,8 @@ export class FSWatcher extends EventEmitter {
   }
 
   emitWithAll(event: EventName, args: EmitArgs) {
-    this.emit(...args);
-    if (event !== EV.ERROR) this.emit(EV.ALL, ...args);
+    this.emit(event, ...args);
```

Comment:
Huh?

Reply:
`args` used to contain the event name too and we basically didn't make use of `event` even though we passed it

I simplified so we use the event name and the args separately

We only had them combined for storing in the pending unlinks map. Everywhere else, we already had the name separately or didn't need it